rpc/client/httpclient.go

```diff
@@ -39,17 +39,14 @@ func NewHTTP(remote, wsEndpoint string) *HTTP {
 	}
 }
 
-func (c *HTTP) _assertIsClient() Client {
-	return c
-}
-
-func (c *HTTP) _assertIsNetworkClient() NetworkClient {
-	return c
-}
-
-func (c *HTTP) _assertIsEventSwitch() types.EventSwitch {
-	return c
-}
+var (
+	// Some compile time assertions to ensure that
```

Comment:
don't think this comment is necessary

Reply:
Roger that, @melekes I'll update that, thanks.